internal/cli/start.go

```diff
@@ -0,0 +1,52 @@
+package cli
+
+import (
+	"flag"
+	"fmt"
+	"log"
+
+	server "github.com/canonical/notary/internal/api"
+	"github.com/canonical/notary/internal/config"
+)
+
+type StartCommand struct {
+	fs         *flag.FlagSet
+	configPath string
+}
+
+func (startCommand *StartCommand) Name() string {
+	return startCommand.fs.Name()
+}
+
+func (startCommand *StartCommand) Init(args []string) error {
+	return startCommand.fs.Parse(args)
+}
+
+func (startCommand *StartCommand) Run() error {
+	if startCommand.configPath == "" {
+		return fmt.Errorf("providing a config file is required")
```

Comment:
I was reading and apparently it's better to use `%w` for errors.
It wraps the error when returning allowing the caller to unwrap it.
https://go.dev/wiki/ErrorValueFAQ

So basically the caller of Run would be able to check which error was returned out of the 4 here.
We probably won't use it because we just are going to exit and log the error whichever that was, therefore, it's fine not to change it, just thought it was interesting to share.

Reply:
Done